src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -112,6 +112,8 @@ export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit,
  */
   @Input()
   public set value(value: any) {
+    console.log('WHAT THE ACTUAL fuck');
```

Comment:
??? :-P

Reply:
hahahaha oh my 😳 I always check for log statements when I'm done, _of course_ I'd miss this one